bip-0065.mediawiki

```diff
@@ -39,6 +39,8 @@ indirectly verify that the desired block height or block time has been reached;
 until that block height or block time has been reached the transaction output
 remains unspendable.
 
+The locktime threshold is 500,000,000. Numbers below this threshold are interpreted
+as block height locktimes. Numbers greater than or equal to this threshold are interpreted as wall clock time locktimes.
```

Comment:
```suggestion
as block height locktimes. Numbers greater than or equal to this threshold are interpreted as wall clock (UNIX timestamp) locktimes.
```

(perhaps also s/numbers/values/ here and in the preceding line, feel free to ignore)